RecoLocalCalo/HcalRecProducers/python/HBHEMethod0Parameters_cfi.py

```diff
@@ -2,7 +2,7 @@
 
 # Configuration parameters for Method 0
 m0Parameters = cms.PSet(
-    firstSample = cms.int32(4),
```

Comment:
@bsunanda : do I understand correctly that this parameter (firstSample) is removed from the config because it is actually not needed by the HBHEPhase1Reconstructor?